mods/ctl/test/commands/apps/deploy.apps.unit.test.ts

```diff
@@ -18,10 +17,14 @@ describe("@fonos/ctl/apps", () => {
 
   beforeEach(() => {
     actionStub = sandbox.stub(cli.action, "start");
-    const app = new App();
-    app.setName("My App");
-    app.setDescription("A test application");
-    app.setCreateTime("January 01, 1970 00:00:00 UTC.");
+    const app = {
+      name:"My App",
+      description: "A test application",
+      createTime: "January 01, 1970 00:00:00 UTC.",
+      updateTime: "January 01, 1970 00:00:00 UTC.",
+      status: "ACTIVE",
+      accessKeyId: "70lkv20G5MCgUIKYJI6Z"
+  }
```

Comment:
Needs linting

Reply:
there is not issue apparently in this case. I ran lint and not issue appears.